mobile/apps/photos/lib/core/configuration.dart

```diff
@@ -704,6 +709,44 @@ class Configuration {
     await _preferences.setBool(hasSelectedAllFoldersForBackupKey, value);
   }
 
+  int? getOnlyNewSinceEpoch() {
```

Comment:
Can we define these new keys in local_settings

Reply:
Done!